doc/user-manual/team.rst

```diff
@@ -3,42 +3,75 @@
 The Agda Team and License
 *************************
 
-Authors:
+Agda 2 was originally written by Ulf Norell,
+partially based on code from Agda 1 by Catarina Coquand and Makoto Takeyama,
+and from Agdalight by Ulf Norell and Andreas Abel.
+
+Agda 2 is currently actively developed mainly by (in alphabetical order):
 
-* Ulf Norell
 * Andreas Abel
+* Guillaume Allais
+* Liang-Ting Chen
+* Jesper Cockx
 * Nils Anders Danielsson
-* Makoto Takeyama
-* Catarina Coquand
+* Ulf Norell
+* Víctor López Juan
```

Comment:
`Víctor López Juan` must be before `Ulf Norell`.

Reply:
Thanks, fixed.